src/lib/common/sol-mainloop-impl-glib.c

```diff
@@ -130,15 +130,20 @@ sol_mainloop_impl_idle_del(void *handle)
     return g_source_remove((uintptr_t)handle);
 }
 
-struct fd_wrap_data {
+struct sol_fd_glib {
```

Comment:
This matches the name used by the posix implementation. Also, this has been re purposed - it has the same initial fields, but it is not used in the same way. It used to be a wrapper, now it's the object returned to the user.